python_legacy/iceberg/api/types/types.py

```diff
@@ -696,6 +696,12 @@ def key_id(self):
     def value_id(self):
         return self.value_field.field_id
 
+    def asMapType(self):
```

Comment:
It looks like these are being converted to snake_case instead. Should this be converted as well?

Reply:
I fully agree with you, but it is part of the [Type interface](https://github.com/apache/iceberg/blob/master/python_legacy/iceberg/api/types/type.py#L66).  I didn't want to change too much in fear of breaking other code depending on this API.  I can make the change if you like, but not sure if it could create side effects.  Probably nothing major since code already using that interface method are always getting `False`...!